$NetBSD: patch-iodev_network_slirp_compat_cc,v 1.1 2015/07/01 13:39:17 ryoon Exp $

(At least NetBSD/amd64 7.99.1 has following problem)
slirp/compat.cc:141:40: error: 'WIFEXITED' was not declared in this scope
slirp/compat.cc:144:35: error: 'WEXITSTATUS' was not declared in this scope

--- iodev/network/slirp/compat.cc~	2014-04-19 02:14:32.000000000 +0900
+++ iodev/network/slirp/compat.cc	2014-12-12 01:01:35.000000000 +0900
@@ -27,6 +27,10 @@
 
 #include "slirp.h"
 
+#ifdef __NetBSD__
+# include "sys/wait.h"
+#endif
+
 #if BX_NETWORKING && BX_NETMOD_SLIRP
 
 void pstrcpy(char *buf, int buf_size, const char *str)
